.github/workflows/cifuzz.yml

```diff
@@ -22,7 +22,8 @@ on:
       - main
 jobs:
   Fuzzing:
-    runs-on: ubuntu-latest
+    # FIXME: Figure out why 32-bit applications fail to run in docker on Ubuntu 24.04.
+    runs-on: ubuntu-22.04
```

Comment:
`cflite_pr.yml` uses basically the same OSS-Fuzz base-builder images and the same runtime so `runs-on` should probably be set to `ubuntu-22.04` there too.

Reply:
I opened https://github.com/google/oss-fuzz/issues/12042

Reply:
It's probably https://github.com/actions/runner-images/issues/9977#issuecomment-2153044449